src/styles/blocks/star.scss

```diff
@@ -0,0 +1,32 @@
+.stars {
+  display: flex;
+
+  &__star {
+    background-image: url(../images/star.svg);
```

Comment:
It seems like you are using a direct URL to the star image here, whereas in lines 14, 18, 22, 26, and 30 you're using a SCSS variable `$star-img`. According to the [Code Style Rules](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), you should be consistent in your approach. If you have a variable for the image, use it consistently across your SCSS.